utils/envvars/parser.go

```diff
@@ -32,13 +32,14 @@ import (
 )
 
 const (
-	defaultSaaSHost = "check.percona.com"
-	envSaaSHost     = "PERCONA_TEST_SAAS_HOST"
```

Comment:
Should it be backward compatible? ie we have new env var, but old one should be supported as well?
Otherwise, it will be hard to test, and it just a good practice - to have grace period and warn when old feature is used.

Reply:
`PERCONA_TEST_*` variables are completely for internal usage. No need to have backward compatibility for them, we just replacing them at all places an once.